### DISCUSSION TITLE
What does Fla in FlaUI mean?

### DISCUSSION THREAD

**Discussion by iyashi39** [FlaUI/FlaUI#576] [Q&A] [answered]
Sorry if this is not the right place to ask but I'm curious what it means and could not find any answers about the name of this library.

**Comment by Roemer** [comment, marked as answer]
Well this is kind of my personal joke. 25 years ago when I participated in a middle-earth (medieval) event, I bought a cape, which was very fluffy (flauschig in German). Since then I named all my personal projects something with "flauschig" or it's abbreviation, Flau. As this has something to do with UI, I just mixed that together: FlaUI

**Reply by iyashi39** [reply]
Thanks for the explanation and for creating this project!
